src/ingestion_data.py

```diff
@@ -4,6 +4,15 @@
 
 from src.graph_model import (Version, EventType, SecurityEvent, FeedBackType, StatusType, EcosystemType)
 from src.parse_datetime import from_date_str, get_date, get_year, get_yearmonth
+from src.config import MAX_STRING_LENGTH
+
+CVE_REGULAR_EXPRESSION = r"CVE-\d{4}-\d{4,}"
+
+
+def get_cves_from_text(data: str) -> set:
```

Comment:
This should be a private function.
```suggestion
def _get_cves_from_text(data: str) -> set:
```

Reply:
done